assets/app/scripts/services/navigate.js

```diff
@@ -48,6 +48,13 @@ angular.module("openshiftConsole")
         $location.path("project/" + encodeURIComponent(projectName) + "/create/next").search("name", name);
       },
 
+      toPodsForDeployment: function(deployment) {
+        $location.url("/project/" + deployment.metadata.namespace + "/browse/pods");
```

Comment:
other places we are using $location.url we don't have the leading slash on /project

i know there were some issues around when we have a context root so should probably be careful here